recipe/assets.php

```diff
@@ -2,13 +2,51 @@
 
 namespace Deployer;
 
-desc('Install bundle\'s web assets under a public directory');
+desc('Run the build script which will build our needed assets.');
 task(
-    'sumo:assets:install',
+    'sumo:assets:fix-node-version',
```

Comment:
Kan het zijn dat deze task `sumo:assets:fix-node-version` nergens wordt aangeroepen?

Reply:
Klopt, ik wou dat nog overleggen met jou. In de assets:install doen we ook al `nvm use` en ik dacht dat die zelf al nvm install ging doen als hij een versie niet had. Deze taak is dus mogelijks overbodig.

Reply:
Ik denk het niet, denk dat als je `nvm use` doet van versie die je niet hebt dat er een foutmelding is. Dus roep die taak maar aan.

Reply:
Staat nu in de default project.php recipe